.github/workflows/build.yml

```diff
@@ -66,5 +66,34 @@ jobs:
                 run: composer test
 
             -
-                name: Run lint container
-                run: (cd src/Bundle/test && bin/console lint:container)
+                name: Run lint container (with all services)
+                run: |
+                    rm -rf src/Bundle/test/var/cache
+                    (cd src/Bundle/test && bin/console lint:container)
+
+            -
+                name: Run lint container (with Swift Mailer)
+                run: |
+                    rm -rf src/Bundle/test/var/cache
+                    mv src/Bundle/test/config/packages/mailer.yaml mailer.yaml
+                    composer remove symfony/mailer --dev
+                    (cd src/Bundle/test && bin/console lint:container)
```

Comment:
if we keep this test suite this way for a moment, I think we have to add `composer require --dev symfony/mailer` at the end

Reply:
I’m not following what it is you’re suggesting to do.  The order of operations for all the Composer commands is fine as is, there is no point in doing a full reset to the starting state at the end of each lint step (everything a step is doing is building off the state the previous step left things in).

Reply:
By that same logic, you can’t rely on the previous steps to have set up the environment correctly either 😉 (if these were parallel or separately running jobs it’d be another story, but it’s a single threaded build).

The only difference in workflows is where certain commands are run, and running less Composer commands overall since each step builds on the previous.